k8s.gcr.io/lib.sh

```diff
@@ -217,6 +248,27 @@ function empower_group() {
         "${bucket}"
 }
 
+# Grant write privileges on a bucket to a group
+# $1: The GCP project
+# $2: The googlegroups group
+# $3: The bucket
+function empower_group_to_bucket() {
```

Comment:
maybe we should rename existing functions like empower_group to empower_group_for_gcr or somethign?

Reply:
Did some renames where I thought it made sense.